string model_def
---
bool result